docs/data/toolpad/components/datepicker/DatePickerDisabled.js

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { DatePicker } from '@mui/toolpad-components';
+import { Stack } from '@mui/material';
+
+const TOOLPAD_PROPS1 = {
+  variant: 'outlined',
+  size: 'small',
+  disabled: true,
+  label: 'Disabled',
+};
+
+const TOOLPAD_PROPS2 = {
```

Comment:
Why these property objects? Can't it be done like in https://github.com/mui/mui-toolpad/pull/2827/files#diff-a961ef6fd4d3a6c4228a46a2b7eb76832b7d49135bc1cc8eef36466f5d1d47bdR8